v3.1/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/vshn/modsecurity-crs:v3.1
+FROM docker.io/owasp/modsecurity-crs:3.3-apache
```

Comment:
The format of allowed_request_content_type inside crs-setup.conf changed. So if we want to override this variable (line 28) we have do adopt it too: 
Also see here: 

- https://github.com/coreruleset/coreruleset/blob/v3.3/dev/CHANGES#L19
- https://github.com/coreruleset/coreruleset/blob/v3.3/dev/crs-setup.conf.example#L431

Reply:
We don't set this variable at all in our image and is not used in the crs-setup.conf at all. So I don't think we need to do anything here. Do you agree?

Reply:
You set it here:
https://github.com/vshn/modsecurity-docker/blob/9f5d49d0e252981220e8f56fd54cb90b9f18c534/v3.1/Dockerfile#L28